packages/cli/src/ceramic-daemon.ts

```diff
@@ -448,6 +444,17 @@ export class CeramicDaemon {
     })
   }
 
+  async getCollection(req: Request, res: Response): Promise<void> {
+    const httpQuery = parseQueryObject(req.query)
+    const defaultPagination = { first: 100 }
```

Comment:
I'm wondering if we should even have this default, rather than simply making it an invalid request to no provide either `first` or `last`?

Reply:
@PaulLeCam I can make no strong argument here. I basically borrowed this from TheGraph default behaviour. Removing it now in favour of an error. It is based on an incoming payload only, so is fine wrt security.